Orange/widgets/visualize/owscatterplotgraph.py

```diff
@@ -790,17 +966,19 @@ def get_coordinates(self):
         x, y = self.jitter_coordinates(x, y)
         return x, y
 
-    def _create_sample(self):
+    def _create_sample(self, mask=None):
         """
         Create a random sample if the data is larger than the set sample size
         """
         self.n_shown = min(self.n_valid, self.sample_size or self.n_valid)
         if self.sample_size is not None \
                 and self.sample_indices is None \
                 and self.n_valid != self.n_shown:
-            random = np.random.RandomState(seed=0)
-            self.sample_indices = random.choice(
-                self.n_valid, self.n_shown, replace=False)
+
+            np_random = np.random.RandomState()
```

Comment:
Why seed=None here? I'd leve it as it was.
Also, why rename? That is just opportunity for more merge conflicts.